### PR TITLE
Geographic Vectors Examples 

### PR DESCRIPTION
**Pull request by munzekm**
Branch to create examples of geographic vectors with pygmt.

### CODE REVIEW

**Review by noorbuchi — CHANGES_REQUESTED** [review]
There are three `Untitled.ipynb` that are not needed. Please make sure to remove those before merging

**Comment by cklima616** [comment]
Fixing the import order will get rid of one of those errors.

**Review by cklima616 — CHANGES_REQUESTED** [review]
Fix the import order and get rid of those Untitled files and it'll be set to merge!

File: examples/tutorials/vectors.py [hunk 1]
```diff
@@ -245,134 +245,145 @@
...
-# Change line attributes
-# ----------------------
-#
-# The line attributes can be set by the ``pen`` parameter. ``pen`` takes a string
-# argument with the optional values *width*,\ *color*,\ *style*.
-#
-# In the example below, the pen width is set to ``5p``, and with ``black`` as the
-# default color and ``solid`` as the default style.
+x = [-114.7420, 44.0682]
+y = [-87.6298, 41.8781]
```
Comment (noorbuchi) [inline examples/tutorials/vectors.py:290]: I think that renaming those variables to `point_1` and `point_2` might be better, in this case, to make it clear that these are not a list of x values and y values similar to how cartesian vectors work

**Review by noorbuchi — CHANGES_REQUESTED** [review]

File: examples/tutorials/vectors.py [hunk 2]
```diff
@@ -245,134 +245,145 @@
...
+data = np.array([x + y])
```
Comment (noorbuchi) [inline examples/tutorials/vectors.py:291]:
```suggestion
point_1 = [-114.7420, 44.0682]
point_2 = [-87.6298, 41.8781]
data = np.array([point_1 + point_2])
```

**Pull request merged**